packer/variables.pkr.hcl

```diff
@@ -37,3 +37,8 @@ variable "zone" {
   type    = string
   default = ""
 }
+
+variable "skip_create_image" {
+  type    = string
```

Comment:
Should this be `bool` instead of `string`?

Reply:
this went though which is concerning, but nice catch